proxy/multiplexer.go

```diff
@@ -83,6 +87,7 @@ func (m *multiplexer) routeMessages(ws *websocket.Conn) {
 
 	// Write messages to backend
 	go func(stop <-chan bool) {
+		c := time.Tick(time.Second * 5)
```

Comment:
@cjellick I was actually thinking about the Tick.  That creates a goroutine, right?  That can never be closed??